parsers/AM.py

```diff
@@ -54,10 +56,14 @@
     "[28]": "sparum2",
 }
 
+REGEX = "[-+]?[.]?[\d]+(?:,\d\d\d)*[\.]?\d*(?:[eE][-+]?\d+)?"
+
 
-def fetch_production(zone_key="AM", session=None, target_datetime=None, logger=None):
+def fetch_production(
+    zone_key="AM", session=None, target_datetime=None, logger=None
+) -> dict:
     r = session or requests.session()
-    url = "http://epso.am/poweren.htm"
+    url = SOURCE
     response = r.get(url)
```

Comment:
`url` isn't used further below

```suggestion
    response = r.get(SOURCE)
```